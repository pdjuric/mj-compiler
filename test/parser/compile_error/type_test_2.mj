program type_test

int A;
A a;
{
    void main() {}
}