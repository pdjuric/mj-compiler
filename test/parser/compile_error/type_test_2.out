ERROR 17:12:54,996 - line   4 - A is not a type